packages/core/src/lib/implementation/read-rc-file.ts

```diff
@@ -7,12 +13,42 @@ import {
 } from '@code-pushup/models';
 import { fileExists, importModule } from '@code-pushup/utils';
 
+function formatErrorPath(errorPath: (string | number)[]): string {
+  return errorPath
+    .map((key, index) => {
+      if (typeof key === 'number') {
+        return `[${key}]`;
+      }
+      return index > 0 ? `.${key}` : key;
+    })
+    .join('');
+}
+
+const coreConfigMessageBuilder: MessageBuilder = issues =>
+  issues
+    .map(issue => {
+      const formattedMessage = red(`${bold(issue.code)}: ${issue.message}`);
+      const formattedPath = formatErrorPath(issue.path);
+      if (formattedPath) {
+        return `Validation error at ${bold(formattedPath)}\n${formattedMessage}\n`;
+      }
+      return `${formattedMessage}\n`;
+    })
+    .join('\n');
```

Comment:
These 2 functions could live in `utils` package, since they are applicable to any Zod schema.